src/Components/Pet.js

```diff
@@ -1,9 +1,66 @@
 import React from 'react';
 import '../Styles/Pet.scss';
+import Header from './Header';
+
+//Posting happens before querying
```

Comment:
Probably want to remove the pseudocode.

Reply:
And done. Stevie cleaned it up and pushed before the merge